datastore/uncached.go

```diff
@@ -137,3 +142,37 @@ func (s *DataStore) GetAllConfigMaps() (runtime.Object, error) {
 func (s *DataStore) GetAllVolumeAttachments() (runtime.Object, error) {
 	return s.kubeClient.StorageV1().VolumeAttachments().List(context.TODO(), metav1.ListOptions{})
 }
+
+// FindARandomReadyNode return a random ready node.
+// Return error if there is no ready node
+func FindARandomReadyNode(ei *longhorn.EngineImage, nodeList *longhorn.NodeList) (string, error) {
```

Comment:
Why do you add the function in datastore pkg? or just no good place to add it?

Just thinking if we should better organize the shared util function/service well in the future. (not a request to change here)

Reply:
cc @jenting @shuo-wu @joshimoo

Reply:
Personally, I would like to move CR CRUD functions into `datastore`. And I will consider this as a kind of `Get` function.

Reply:
Agree with Shuo